tests/kernel/common.cl

```diff
@@ -8,7 +8,7 @@
 #endif
 
 #undef __IF_INT64
-#if defined(__opencl_c_int64) || defined(cl_khr_int64)
```

Comment:
I'm missing an explanation for this change. AFAICT the `__opencl_c_int64` is an actual language feature in the official OpenCL 3.0 Specification, and `cl_khr_int64` is a made-up "extension" used in past by PoCL to signal & check 64bit integer support. So... this seems going in the wrong direction, and also no mention why is this required. Could you add an explanatory comment.